code/game/machinery/autolathe.dm

```diff
@@ -37,10 +37,10 @@
 	wires = new(src)
 
 	for(var/Name in name_to_material)
-		if(Name in stored_material)
+		if(Name in materials)
 			continue
 
-		stored_material[Name] = 0
+		materials[Name] = 0
 		storage_capacity[Name] = 0

```

Comment:
honestly i have no idea, that's just how the other fabrication machines do it and i'm too scared to change it in case it breaks something

if i had to take a guess, it's so nonstandard materials that probably shouldn't be in there (e.g. gold) don't fuck things up if somehow they end up in there

Reply:
Storage capacity is updated according to matterbin capacity for all things in `storage_capacity` by the following:
```
	for(var/mat_name in storage_capacity)
		storage_capacity[mat_name] = mb_rating * 25000
```
https://github.com/PolarisSS13/Polaris/pull/9046/files#diff-b7b4916a8ea53d9762bd6cb0cc4251d0649d08e548b5157f511604542a6741b5L359-L365
So I don't see how that could be broken.

> that's just how the other fabrication machines do it

Cargo cult is a bad excuse. I am generally an advocate for fucking around and finding out if something is actually broken.